deploy/helm/kuberhealthy/values.yaml

```diff
@@ -93,7 +93,7 @@ check:
       HOSTNAME: "kubernetes.default"
     nodeSelector: {}
   podRestarts:
-    enabled: false
```

Comment:
I think you should switch these back to false -- I'm not sure if these checks are considered `out-of-the-box` yet

Reply:
I think you're right I might of included them because I was generating the new templates.